### PR TITLE
[FSTORE-1346] Upgrade doc dependencies, contributing.md, mkdocs workflows

### PR DESCRIPTION
**Pull request by vatj**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -118,62 +118,62 @@ On docs.hopsworks.ai we implement the following versioning scheme:
 
 ###### Build Instructions
 
-4. For this you can either checkout and make a local copy of the `upstream/gh-pages` branch, where
```
Comment (robzor92) [inline CONTRIBUTING.md:121]: The section below here does not render correctly

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -118,62 +118,62 @@ On docs.hopsworks.ai we implement the following versioning scheme:
 
 ###### Build Instructions
 
-4. For this you can either checkout and make a local copy of the `upstream/gh-pages` branch, where
```
Comment (robzor92) [inline docs/CONTRIBUTING.md:121]: The section below here does not render correctly

**Review by robzor92 — CHANGES_REQUESTED** [review]

**Review by robzor92 — CHANGES_REQUESTED** [review]
mkdocs-release workflow doesn't work and should we not also add this change? https://github.com/logicalclocks/feature-store-api/pull/1275/files

**Comment by vatj** [comment]
Added the changes, still testing it

**Comment by vatj** [comment]
Merged into FSTORE-1371

**Pull request closed without merging**